munit/shared/src/main/scala-3/munit/internal/MacroCompat.scala

```diff
@@ -10,8 +10,8 @@ object MacroCompat {
     inline implicit def generate: Location = ${ locationImpl() }
   }
 
-  def locationImpl()(given qctx: QuoteContext): Expr[Location] = {
-    import qctx.tasty.{_, given}
+  def locationImpl()(using qctx: QuoteContext): Expr[Location] = {
+    import qctx.tasty.{_, given _}
```

Comment:
If I'm not mistaken (by reading the release notes), this can just be

```suggestion
    import qctx.tasty._
```

Reply:
I just tried applying this change and it did not compile
```
.
[error] -- [E008] Member Not Found Error: /Users/lgeirsson/dev/munit/munit/shared/src/main/scala-3/munit/internal/MacroCompat.scala:15:28
[error] 15 |    val path = rootPosition.sourceFile.jpath.toString
[error]    |               ^^^^^^^^^^^^^^^^^^^^^^^
[error]    |               value sourceFile is not a member of qctx.tasty.Position
[error] -- [E008] Member Not Found Error: /Users/lgeirsson/dev/munit/munit/shared/src/main/scala-3/munit/internal/MacroCompat.scala:16:33
[error] 16 |    val startLine = rootPosition.startLine + 1
[error]    |                    ^^^^^^^^^^^^^^^^^^^^^^
[error]    |value startLine is not a member of qctx.tasty.Position, but could be made available as an extension method.
[error]    |
[error]    |The following import might fix the problem:
[error]    |
[error]    |  import qctx.tasty.positionOps
[error]    |
[error] -- Error: /Users/lgeirsson/dev/munit/munit/shared/src/main/scala-3/munit/internal/MacroCompat.scala:17:32
[error] 17 |    '{ new Location(${Expr(path)}, ${Expr(startLine)}) }
[error]    |                                ^
[error]    |ambiguous implicit arguments: both method DoubleIsLiftable in object Liftable and method LongIsLiftable in object Liftable match type quoted.Liftable[String] of an implicit parameter of method apply in object Expr
[error] -- Error: /Users/lgeirsson/dev/munit/munit/shared/src/main/scala-3/munit/internal/MacroCompat.scala:17:52
[error] 17 |    '{ new Location(${Expr(path)}, ${Expr(startLine)}) }
[error]    |                                                    ^
[error]    |ambiguous implicit arguments: both method DoubleIsLiftable in object Liftable and method LongIsLiftable in object Liftable match type quoted.Liftable[Int] of an implicit parameter of method apply in object Expr
[error] -- [E008] Member Not Found Error: /Users/lgeirsson/dev/munit/munit/shared/src/main/scala-3/munit/internal/MacroCompat.scala:26:23
[error] 26 |    val source = value.unseal.pos.sourceCode
[error]    |                 ^^^^^^^^^^^^
[error]    |value unseal is not a member of quoted.Expr[T], but could be made available as an extension method.
[error]    |
[error]    |The following import might fix the problem:
[error]    |
[error]    |  import qctx.tasty.QuotedExprOps.unseal
[error]    |
[error] -- Error: /Users/lgeirsson/dev/munit/munit/shared/src/main/scala-3/munit/internal/MacroCompat.scala:28:30
[error] 28 |    '{ new Clue(${Expr(source)}, $value, ${Expr(valueType)}) }
[error]    |                              ^
[error]    |ambiguous implicit arguments: both method DoubleIsLiftable in object Liftable and method LongIsLiftable in object Liftable match type quoted.Liftable[String] of an implicit parameter of method apply in object Expr
[error] 6 errors found
[error] (munitJVM / Compile / compileIncremen
```

Reply:
ok, thanks for trying!